docs/source/models/concept_models/vivarium_mncnh_portfolio/anemia_component/module_document.rst

```diff
@@ -44,9 +44,53 @@ Anemia Module
 
 This document is the page for the anemia YLDs module of the MNCNH Portfolio simulation.
 The anemia module will process information related to simulants' hemoglobin exposure
-throughtout the course of the simulation and calculate corresponding anemia exposure
+throughout the course of the simulation and calculate corresponding anemia exposure
 and YLDs.
 
+Note that the simulation implementation of anemia YLDs as it relates to the 
+:ref:`hemoglobin exposure modules <2024_vivarium_mncnh_portfolio_hemoglobin_module>`
+has been performed slightly differently than written on these pages. The implementation
+strategy can be summarized as follows:
+
+  In order to keep single definition of hemoglobin exposure throughout entire sim (as opposed to separate measures such as "hemoglobin at the start of pregnancy" and "hemoglobin at the end of pregnancy" as documented), we have multiple hemoglobin-related timesteps and update hemoglobin exposure and assess anemia YLDs across those timesteps rather than assess anemia YLDs as a lump sum from different hemoglobin values at different timepoints. Anemia YLDs are estimated according to the following steps:
```

Comment:
```suggestion
  In order to keep a single definition of hemoglobin exposure throughout entire sim (as opposed to separate measures such as "hemoglobin at the start of pregnancy" and "hemoglobin at the end of pregnancy" as documented), we have multiple hemoglobin-related timesteps and update hemoglobin exposure and assess anemia YLDs across those timesteps rather than assess anemia YLDs as a lump sum from different hemoglobin values at different timepoints. Anemia YLDs are estimated according to the following steps:
```